meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -108,7 +108,12 @@ function log {
 }
 
 function runPreHacks {
-    local _boot_mountpoint="$(grep $(blkid | grep resin-boot | cut -d ":" -f 1) /proc/mounts | cut -d ' ' -f 2)"
+    # If there is no blkid assume boot partition in /boot
+    if which blkid &> /dev/null; then
+        local _boot_mountpoint="$(grep $(blkid | grep resin-boot | cut -d ":" -f 1) /proc/mounts | cut -d ' ' -f 2)"
+    else
+        local _boot_mountpoint="/boot"
+    fi
```

Comment:
The problem can get a little more complex as I don't know when blkid was introduced in the hostOS. So if I do it by version I have to be sure that after boot partition was moved to mnt/boot, blkid was always persistent in the fs. On the other hand we can simplify this by doing umount on boot and mnt/boot if we don't have blkid. This will be a decent hack until we will migrate all the old devices.